src/Controller/Base.php

```diff
@@ -378,6 +378,12 @@ protected function getRepository($repository)
      */
     protected function getContent($textquery, $parameters = [], &$pager = [], $whereparameters = [])
     {
+        $queryVersion = $this->getOption('compatibility/setcontent_version', 1);
```

Comment:
Unless I am missing something we've done, the first parameter is missing `general/`, as is [the two here](https://github.com/bolt/bolt/blob/release/3.3/src/Controller/Base.php#L84-L91)

Testing without defaults, or debug assignments, shows this to be the case :smile: